content/index.html

```diff
@@ -14,9 +14,9 @@ <h3>Getting started</h3>
   </div>
   <div class="tc card w-100 w-30-ns pa3 mb3 br2"">
     <i class="db f2 fa fa-user-check"></i>
-    <h3>Personal plan guide</h3>
-    <p>Make the most of your personal plan subscription.</p>
-    <a href="/articles/getting-started-personal-plan">Personal plan</a>
+    <h3>Solo plan guide</h3>
```

Comment:
I know this was here before as "Personal plan guide" but I think I'd rather replace it with something that is not plan-specific, or remove it altogether.

Reply:
I'll remove it for now.